pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+# Description
```

Comment:
Please remove this file. Ayo checked-in the file to [.github repo](https://github.com/benchpress-team/.github/blob/main/pull_request_template.md).